spec/migrations/20171106143927_downcase_resource_group_ems_ref_for_azure_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_migration
+
+describe DowncaseResourceGroupEmsRefForAzure do
+  let(:resource_group_stub) { migration_stub :ResourceGroup }
+
+  let!(:uppercase_stub) do
+    resource_group_stub.create!(:name => 'UPPER', :ems_ref => '/subscriptions/xyz/resourceGroups/UPPER')
```

Comment:
Since the resource_group table uses STI, add a type here because a "real" record will always have a type (the subclass fully qualified name, eg. ManageIQ::Providers::Azure::ResourceGroup - I think that's the name, but get a real one to be sure).  It really doesn't even matter what the type is as long and it's not "ResourceGroup", and once you add that the tests will fail.  To get them green, you have to do what I said [up there](https://github.com/ManageIQ/manageiq-schema/pull/123/files#r149191728).